### PR TITLE
Add u-band SLR measurements

### PR DESCRIPTION
**Pull request by erykoff**
This also does a bunch of smaller fixups including changing the directory variable names to make it clear(er) which is which.

### CODE REVIEW

File: python/lsst/the/monster/measure_uband_slr_colorterm.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+        List of HTM pixel ids to load.
+    testing_mode : `bool`, optional
+        Running in "testing" mode?
+
+    Returns
+    -------
+    gaia_stars_all : `astropy.table.Table`
+        Table of stars, with fluxes filled in.
+    """
+    u_slr_bands = ["g", "r"]
```
Comment (psferguson) [inline python/lsst/the/monster/measure_uband_slr_colorterm.py:64]: Do we want this hardcoded? or we could pass the uband_ref_info.get_color_bands("u") result/refcatInfo

**Review by psferguson — APPROVED** [review]
Looks good, left some minor comments

File: bin.src/measure_gaiaxpu_colorterm.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+from lsst.the.monster import GaiaXPuSplineMeasurer
+
+
+measurer = GaiaXPuSplineMeasurer()
+measurer.measure_spline_fit(bands=["u"])
```
[the remark this answers is not in the document]
Comment (erykoff) [reply bin.src/measure_gaiaxpu_colorterm.py:7]: I wanted the name to make it clear that this is specifically about XP u-band.

File: python/lsst/the/monster/measure_uband_slr_colorterm.py [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+from .refcats import GaiaDR3Info, GaiaXPInfo, GaiaXPuInfo, DESInfo, SkyMapperInfo, PS1Info, VSTInfo
+from .splinecolorterms import ColortermSplineFitter, ColortermSpline, MagSplineFitter
+from .utils import read_stars
+from .refcats import RefcatInfo
+
+
+__all__ = [
+    "UbandSLRSplineMeasurer",
+    "read_uband_combined_catalog",
+]
```
[the remark this answers is not in the document]
Comment (erykoff) [reply python/lsst/the/monster/measure_uband_slr_colorterm.py]: I don't think it matters, but sure!

**Comment by psferguson** [comment]
Already approved, but changes look good to me!

**Pull request merged**